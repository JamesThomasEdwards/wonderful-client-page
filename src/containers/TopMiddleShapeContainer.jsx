// React;
import React, { Component } from 'react';
// Files;
import DiamondAngleLeft from '../components/shapes/DiamondAngleLeft.jsx';
import DiamondAngleRight from '../components/shapes/DiamondAngleRight.jsx';

export default class TopMiddleShapeContainer extends Component {
    state = {
        animate: false
    };
    render() {
        return (
            <div className="top-middle-shape-container">
                <DiamondAngleLeft
                    styleImage="diamond-angle-left__man-with-beard" />
                <DiamondAngleRight
                    styleImage="diamond-angle-right__girl-with-darkhair" />
            </div>
        );
    };
};