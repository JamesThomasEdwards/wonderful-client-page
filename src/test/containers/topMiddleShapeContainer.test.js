// React;
import React from 'react';
// Files;
import TopMiddleShapeContainer from '../../containers/TopMiddleShapeContainer.jsx';
// Testing
import ReactShallowRenderer from 'react-test-renderer/shallow';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

const props = 0;
const wrapper = shallow(<TopMiddleShapeContainer {...props} />)

test('should render App component correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<TopMiddleShapeContainer />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});

test('initial state should be false', () => {
    expect(wrapper.state()).toEqual({ animate: false });
});