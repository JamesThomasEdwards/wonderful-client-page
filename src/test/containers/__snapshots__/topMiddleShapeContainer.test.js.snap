// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component correctly 1`] = `
<div
  className="top-middle-shape-container"
>
  <DiamondAngleLeft
    styleImage="diamond-angle-left__man-with-beard"
  />
  <DiamondAngleRight
    styleImage="diamond-angle-right__girl-with-darkhair"
  />
</div>
`;
